c2corg_ui/static/partials/cards/articles.html

```diff
@@ -23,12 +23,12 @@
       </span>
 
       <span ng-if="cardCtrl.remainingActivities" ng-repeat="activity in ::doc.activities | limitTo: (4 - doc.activities.length)"
-            class="activity icon-{{::activity}}" uib-tooltip="{{ cardCtrl.translate(activity)}}">
+            class="activity icon-{{::activity}}" uib-tooltip="{{cardCtrl.translate(activity)}}" tooltip-append-to-body="true">
       </span>
     </div>
 
     <div class="quality" ng-if="::doc['quality']" ng-class="::doc['quality']"
-         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-placement="left">
+         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-append-to-body="true">
```

Comment:
yep! Apparently everywhere where is a translate there should be two-way binding, if not it will only be translated once and will not react on lang change (just tested).